hibike/hibike_message.py

```diff
@@ -333,6 +324,17 @@ def parse_bytes(bytes):
     return None
   return HibikeMessage(messageID, payload)
 
+# Sends this message
+# Computes the checksum
+# Then sends each byte of the message, and finally sends the checksum byte
+def send(serial_conn, message):
```

Comment:
I moved it closer to the read function. This function is meant to use a serial object. There's a separate `hid_send` function for sending over hid. I kept the function name as "send" so it doesn't break any python program that uses this.